dso-l2/src/main/java/com/tc/l2/state/StateManagerImpl.java

```diff
@@ -410,7 +410,7 @@ private synchronized void handleStartElectionRequest(L2StateMessage msg) throws
       groupManager.sendTo(msg.messageFrom(), abortMsg);      
     } else if (!electionMgr.handleStartElectionRequest(msg)) {
 //  another server started an election.  Unclear which server is now active, clear the active and run our own election
-      startElectionIfNecessary(syncdTo);
+      startElectionIfNecessary(ServerID.NULL_ID);
```

Comment:
I think this is wrong.  we need to start election in this case and it will not happen with this parameter.